src/Event/Event.php

```diff
@@ -89,6 +94,29 @@ public function __get($attribute)
         if ($attribute === 'name' || $attribute === 'subject') {
             return $this->{$attribute}();
         }
+        if ($attribute === 'data') {
+            return $this->_data;
+        }
+        if ($attribute === 'result') {
+            return $this->_result;
+        }
+    }
+
+    /**
+     * Provides backward compatibility for write access to data and result properties.
+     *
+     * @param string $attribute Attribute name.
+     * @param mixed $value The value to set.
+     * @return void
```

Comment:
I had these trigger_errors before that made this make sense. I'll switch the properties back to public and label them deprecated, but I have to say using the trigger_errors made it real easy to find where the public methods were being used in all the tests. Most of the IDE usage findings didn't work because of missing type hinting.